src/plugins/condition/components/Condition.vue

```diff
@@ -212,7 +212,8 @@ export default {
             domainObject: this.domainObject,
             currentCriteria: this.currentCriteria,
             expanded: true,
-            selectedOutputKey: '',
+            trigger: 'all',
+            selectedOutputSelection: '',
```

Comment:
Is `stringOutputField` used anywhere? If not, please remove it.

Reply:
done